feel/feelfit/fitdiff.hpp

```diff
@@ -0,0 +1,226 @@
+            M_interpolator(expr.interpolator())
+        {
+        }
+
+        // linear form
+        tensor( this_type const& expr,
+                Geo_t const& geom, Basis_i_t const& fev )
+            :
+            M_tensor_expr( expr.expression(), geom, fev ),
+            M_interpolator(expr.interpolator())
+        {
+        }
+
+        // evaluation
+        tensor( this_type const& expr, Geo_t const& geom )
+            :
+            M_tensor_expr( expr.expression(), geom ),
+            M_interpolator(expr.interpolator())
+        {
+        }
+
+        // IM = 
+        template<typename IM>
+        void init( IM const& im )
+        {
+            M_tensor_expr.init( im );
+        }
+
+        // precompute the part in the bilinear form
+        void update( Geo_t const& geom, Basis_i_t const& /*fev*/, Basis_j_t const& /*feu*/ )
+        {
+            update(geom);
+        }
+        // precompute the part in the linear form
+        void update( Geo_t const& geom, Basis_i_t const& /*fev*/ )
+        {
+            update(geom);
+        }
+        // precompute the part in the evaluate part
+        void update( Geo_t const& geom )
+        {
+            M_tensor_expr.update( geom );
+        }
+        // precompute the part in the evaluate part for a face
+        void update( Geo_t const& geom, uint16_type face )
+        {
+            M_tensor_expr.update( geom, face );
+        }
+
+        // i : local index of basis test function
+        // j : local index of basis trial function
+        // c1 : id x, y  z of the first component (= 0 in scalar case)
+        // c2 : id x, y  z of the second component (= 0 in scalar and vectorial case)
+        // q : id of the current point in the geomap (basically, quadrature point or interpolation point)
+        
+        //local assembling - bilinear form a(i,j) 
+        value_type
+        evalij( uint16_type i, uint16_type j ) const
+        {
+            return 1.; //M_tensor_expr.evalij( i, j );
+        }
+
+        value_type
+        evalijq( uint16_type /*i*/, uint16_type /*j*/, uint16_type c1, uint16_type c2, uint16_type q ) const
+        {
+            return evalq( c1, c2, q );
+        }
+
+        value_type
+        evaliq( uint16_type /*i*/, uint16_type c1, uint16_type c2, uint16_type q ) const
+        {
+            return evalq( c1, c2, q );
+        }
+
+        value_type
+        evalq( uint16_type c1, uint16_type c2, uint16_type q ) const
+        {
+            // here : call the interpolator
+            //return M_interpolator(M_tensor_expr.evalq(c1, c2, q));
+            return M_interpolator->diff(M_tensor_expr.evalq(c1, c2, q));
+        }
+
+    private:
+        tensor_expr_type M_tensor_expr;
+        Interpolator* M_interpolator;
+    };
+    /// end of tensor
+        private:
+            mutable expression_type  M_expr;
+            Interpolator*  M_interpolator;
+    };
+
+/**
+ * \brief FitDiff
```

Comment:
actually \brief is not really necessary if you have indeed a first line with a brief statement